draft-ietf-httpbis-priority.md

```diff
@@ -280,9 +280,11 @@ priority parameters is encoded as a Structured Fields Dictionary (see
 
 This document defines the urgency(`u`) and incremental(`i`) parameters. When
 receiving an HTTP request that does not carry these priority parameters, a
-server SHOULD act as if their default values were specified. Note that handling
-of omitted parameters is different when processing an HTTP response; see
-{{merging}}.
+server SHOULD act as if their default values were specified.
+
+An intermediary that forwards requests and response can combine signals from
+both; see {{merging}}. Note that omission of parameters in responses is handled
+differently from omission in requests.
```

Comment:
```suggestion
An intermediary can combine signals from request and response that it forwards.
Note that omission of parameters in responses is handled differently from
omission in requests; see {{merging}}.
```

This might be less confusing, as by definition, an intermediary forwards messages.

Reply:
subject verb agreement; but this is an improvement
```suggestion
An intermediary can combine signals from requests and responses that it forwards.
Note that omission of parameters in responses is handled differently from
omission in requests; see {{merging}}.
```

Reply:
Agreed, thanks both!